### PR TITLE
Feature/more-ui

### PR DESCRIPTION
**Pull request by Zer0-bit**

### CODE REVIEW

File: webserver/web-interface/src/components/alert/alert.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        // Define your border colors based on the level value
+        if (level === 'INFO') {
+          return '#178bca';
+        }
+        if (level === 'WARN') {
+          return 'orange';
+        }
+        if (level === 'ERROR') {
+          return '#FF5733';
+        }
```
Comment (ckifer) [inline webserver/web-interface/src/components/alert/alert.tsx:48]: nit: maybe this is easier as an object map of level to color string?

File: webserver/web-interface/src/pages/home/Home.jsx [hunk 2]
```diff
@@ -81,8 +82,24 @@ function Home() {
...
+    const targetBox = document.getElementById('targetBox');
+    const currentValue = parseInt(targetBox.value);
+    const newValue = currentValue + 1;
+    targetBox.value = newValue.toString();
+
+  }
+
   return (
     <Container sx={{ pt: theme.spacing(2), gap: '0px' }}>
+      <ShowAlert level='INFO' text='Welcome home motherfucker \_O_/' />
```
Comment (ckifer) [inline webserver/web-interface/src/pages/home/Home.jsx:102]: nice

File: webserver/web-interface/src/pages/home/Home.jsx [hunk 3]
```diff
@@ -81,8 +82,24 @@ function Home() {
...
+    targetBox.value = newValue.toString();
+  }
+
+  function handleAdd() {
+    const targetBox = document.getElementById('targetBox');
+    const currentValue = parseInt(targetBox.value);
+    const newValue = currentValue + 1;
+    targetBox.value = newValue.toString();
+
+  }
```
Comment (ckifer) [inline webserver/web-interface/src/pages/home/Home.jsx:98]: this is a vanilla JS approach to things. We shouldn't need to use `getElementById` unless we're escape hatching from React.

Instead the target value can be state defaulted to `93` and the value can be interpolated and added to below.

This then just becomes 
```jsx
const [targetValue, setTargetValue] = useState(93);

function handleAdd() {
  setTargetValue(oldValue => oldValue + 1);
}

function handleRemove() {
  setTargetValue(oldValue => oldValue - 1);
}
```

File: webserver/web-interface/src/pages/home/Home.jsx [hunk 4]
```diff
@@ -123,11 +140,11 @@ function Home() {
               justifyContent: 'center', alignItems: 'center', display: 'flex', border: `0px solid ${theme.palette.divider}`, position: 'relative', borderRadius: '16px', width: '100%', padding: '10px', gap: '25px',
             }}
             >
-              <TextField variant="standard" sx={{ width: '10ch' }} id="outlined-read-only-input" label="Target" defaultValue="93C" InputProps={{ readOnly: true }} />
-              <Fab color="primary" aria-label="add">
+              <TextField variant="standard" sx={{ width: '10ch' }} id="targetBox" label="Target (°C)" value="93" InputProps={{ readOnly: true }} />
```
Comment (ckifer) [inline webserver/web-interface/src/pages/home/Home.jsx:143]: from above this becomes:

`value={targetValue}`

File: webserver/web-interface/src/pages/home/Home.jsx (hunk 3 above)
Comment (Zer0-bit) [reply webserver/web-interface/src/pages/home/Home.jsx:70]: This way it looks nicer and better indeed, the current value handling ain't what's needed, i was just mocking smth there until  the actual target var is dropped in there and handled dynamically between esp and bp

**Pull request merged**